app/services/notifications/dead_link.rb

```diff
@@ -0,0 +1,28 @@
+module Notifications
+  class DeadLink
+    def initialize(flag)
+      @flag = flag
+    end
+
+    def self.call(flag)
+      new(flag).call
+    end
+
+    def call
+      FlagNotification.with(flag: flag, message: message.content, url: message.url)
```

Comment:
I'd be tempted to inline this in Flags::NotifyUser service for now and keeping the notifications namespace just for discord notifications for the moment. We probably will need to get more specific about the name spacing for notifications in the future since we have two distinct types with discord and internal notifications.